### DISCUSSION TITLE
phoenix.conf

### DISCUSSION THREAD

**Discussion by tipogi** [ACINQ/phoenixd#29] [Q&A] [answered]
Is there some source that I can find which are the parameters for the `~/.phoenix/phoenix.conf` conf file? I would like to spin up a container in testnet?

**Comment by pm47** [comment, marked as answer]
All the options shown in `./phoenixd --help` can be set in `phoenix.conf`, just remove the leading `--`.
For example:
```
chain=testnet
auto-liquidity=10m
http-bind-port=8080
http-password=abcdef
```
